packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -270,6 +280,27 @@ class AndroidCameraCameraX extends CameraPlatform {
         });
   }
 
+  /// Captures an image and returns the file where it was saved.
+  ///
+  /// [cameraId] is not used.
+  @override
+  Future<XFile> takePicture(int cameraId) async {
```

Comment:
@xizhang this is the Dart implementation for taking a picture. Do you have any feedback on binding the `ImageCapture` use case to take picture, and then, immediately unbinding it, as seen below? Wondering if this is best performance-wise. cc @gmackall

Reply:
As I mentioned in the sync meeting. We should leave ImageCapture always bound.

I think something like this would work: Preview, ImageAnalysis and ImageCapture are always bound, since the combination is always supported by the camera hardware. VideoCapture is lazy loaded: it's only bound when the user start recording Video. 

The reason to special case VideoCapture is that VideoCapture is not always supported by camera hardware. When the device doesn't support it, CameraX supports it with OpenGL which will introduce an additional cost. We should avoid that cost if necessary.